### ISSUE TITLE
Cant log into nexus - Windows 10.

### ISSUE THREAD

**Issue by HyperSlayer72** [TanninOne/modorganizer#847]
This has happened to all of my friends as I have tried to log them into the nexus.  Not sure why this is happening.

**Comment by TanninOne** [comment]
Mo version? Any messages in the log view?

**Comment by HyperSlayer72** [comment]
sorry for late responce, mo version 1.3.11 and not sure what you mean by log view, if you mean the window on the bottom the mod then nothing error wise is appearing.

**Comment by Bauxite** [comment]
This is not an MO issue, it's a server issue with Nexus.

01:02:09 [D] login failed: Error downloading http://nmm.nexusmods.com/skyrim/Sessions/?Validate - server replied: Gateway Time-out

http://nmm.nexusmods.com/ is returning a 504 Gateway Time-out error.

**Comment by GrantSP** [comment]
Agree with @Bauxite that the error is Nexus side as it has just hit me. Curiously though my 1.3.11 install fails miserably but my MO2 install shows no error message and tells me I'm logged in, although I'm not.
I just hope it isn't a change in Nexus cookie format or anything like that. If so that may require a change to some code. Maybe the American holiday is causing an overrun of users on Nexus?

**Comment by Bauxite** [comment]
It appears to be just a traffic issue with the login servers, not a configuration change.  NMM is also failing, with the error "Login error: Cannot reach the Nexus login server.  Either your firewall is blocking NMM or the login server is down."

**Comment by HyperSlayer72** [comment]
I would agree, ill check if its a firewall issue, if not then I am not sure if it is a server issue with nexus seeing as I never had issues although anyone I install mo for is having this issue.  Will report back.
